### DISCUSSION TITLE
How to change git repo URL after bootstrap

### DISCUSSION THREAD

**Discussion by sushant08** [fluxcd/flux2#1710] [Q&A]
I have a well running flux2 setup with gitlab as my repo, everything is working fine but i need to update my existing gitlab repo dns/url. Can someone help me in how to change url at flux2 side without breaking things. I see in gitrepositories CRD is one place where i need to update. 

Thanks

**Comment by stefanprodan** [comment]
Rerun `flux bootstrap` for the new URL using the same path as before.

**Reply by sushant08** [reply]
Thanks but is there any other method then bootstrap like updating url in  `GitRepository` CRD

**Reply by stefanprodan** [reply]
You need to change it in Git, then apply the GitRepository object on the cluster manually with `kubectl apply -f flux-system/gotk-sync.yaml`.
